eo2js/src/commands/link.js

```diff
@@ -56,18 +56,21 @@ const link = function(options) {
     path.resolve(project, 'package.json'),
     JSON.stringify(pckg(options))
   )
-  execSync('npm install', {cwd: project})
+  const packageLockPath = path.resolve(project, 'package-lock.json')
```

Comment:
@HaidarJbeily7 let's rename this variable to some single word one: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

After this we're good to merge

Reply:
yup done! Thanks for this article.